scripts/pipeline/symlink_fasta.py

```diff
@@ -0,0 +1,67 @@
+#!/usr/bin/env python3
+
+# See the NOTICE file distributed with this work for additional information
+# regarding copyright ownership.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Script to create and optionally cleanup symlinks in a central location"""
+
+import argparse
+import glob
+import os
+import re
+import sys
+
+from pathlib import Path
+
+parser = argparse.ArgumentParser()
+parser.add_argument('-c', '--cleanup_symlinks', action='store_true')
+parser.add_argument('-s', '--symlink_dir')
+group = parser.add_mutually_exclusive_group(required=True)
+# Only one of target_dir or target_file can be specified
+group.add_argument('-d', '--target_dir')
+group.add_argument('-t', '--target_file')
+opts = parser.parse_args(sys.argv[1:])
+
+target_dir = opts.target_dir
+symlink_dir = opts.symlink_dir
+target_file = opts.target_file
```

Comment:
This is just a suggestion to avoid the use of `os.path.join` later on:
```suggestion
target_dir = Path(opts.target_dir)
symlink_dir = Path(opts.symlink_dir)
target_file = Path(opts.target_file)
```